### PR TITLE
[WIP] add CI for OSX

### PR DESCRIPTION
**Pull request by johnnychen94**
It's highly possible this PR will fail since I don't quite understand the GUI part

let's see how it works.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,4 +24,11 @@ notifications:
 script:
     - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
     - julia -e 'using Pkg; Pkg.clone(pwd()); Pkg.build("ImageView")'
-    - xvfb-run julia -e 'using Pkg; Pkg.test("ImageView")'
+    - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then
+          ( sudo Xvfb :99 -ac -screen 0 1024x768x8; echo ok )& ;
+      else
```
Comment (timholy) [inline .travis.yml]: Why is the `else` here?

Comment (johnnychen94) [reply .travis.yml]: it's a mistake 😄

**Comment by timholy** [comment]
Maybe pause on making further commits to this PR for a little while? At this point my personal Appveyor queue is several hours long, and I am a bit worried that its "cancel automatically if there is a newer pending build" mode could stomp on #171. It may not kick in for separate PRs, but I am not certain of that.

**Comment by johnnychen94** [comment]
I'm not familiar with bash and GUI 😂 This's the only way I can run a test.

**Comment by timholy** [comment]
This is fair game again!

**Comment by johnnychen94** [comment]
emmm is there any frequency limitation on travis? @timholy

**Comment by timholy** [comment]
I don't think so, but something seems wrong. (I expected closing and reopening to fix the problem, but it didn't.) Probably the easiest thing to do would be to rebase this on top of current master and submit a new PR.

**Pull request closed without merging**